app/src/main/java/com/keylesspalace/tusky/util/LinkHelper.kt

```diff
@@ -281,6 +281,10 @@ fun Context.openLink(url: String) {
  */
 private fun openLinkInBrowser(uri: Uri?, context: Context) {
     val intent = Intent(Intent.ACTION_VIEW, uri)
+
+    // Makes sure the Intent opens in the browser instead of something like Mastodon Redirect.
+    intent.selector = Intent(Intent.ACTION_VIEW, Uri.parse("https://"))
```

Comment:
A blank-URL selector guarantees the link won't accidentally open in an app that supports the domain (e.g., Mastodon Redirect, which could cause a launch loop).